lib/cli.js

```diff
@@ -9,11 +9,14 @@ class ProcessorEmitter extends EventEmitter {}
 const processorEmitter = new ProcessorEmitter()
 
 const converter = require('./converter.js')
+// we look forward to the CLI having Invoker.readFromStdin()
+// https://github.com/Mogztter/asciidoctor-pdf.js/pull/158#issuecomment-577788688
+const stdin = require('./stdin')
 
 async function convertFiles (files, argv, options, verbose, preview) {
   for (const file of files) {
     if (verbose) {
-      console.log(`converting file ${file}`)
+      console.log(`converting file ${file.contents ? '-' : file.path}`)
```

Comment:
It's working but in theory we could have a file with a contents and a path, so I would write:

```suggestion
      console.log(`converting file ${file.path || '-'}`)
```